src/menu/menu.ts

```diff
@@ -66,7 +69,8 @@ export class NgAisMenu extends BaseWidget {
   @Input() public showMore?: boolean;
   @Input() public limit?: number;
   @Input() public showMoreLimit?: number;
-  @Input() public sortBy?: string[] | ((item: object) => number);
+  @Input()
+  public sortBy?: FacetSortByStringOptions[] | ((item: object) => number);
```

Comment:
The signature of the function should be more like the one defined inside HierarchicalMenu:

https://github.com/algolia/angular-instantsearch/blob/3f79c4b9864f10061a5ff3c9beb707e4496e0c42/src/hierarchical-menu/hierarchical-menu.ts#L53

Reply:
Yes! I was going to add that then I refrained it looked like I was doing too many changes.
Since you're asking for it I'll add it.

Reply:
done 735ac9f